truffleruby/src/main/java/org/truffleruby/Main.java

```diff
@@ -139,19 +139,15 @@ public static void main(String[] args) {
                         Log.performanceOnce("this JVM does not have the Graal compiler - performance will be limited - see doc/user/using-graalvm.md");
                     }
 
+                    final String bootCode;
                     if (config.shouldUsePathScript()) {
                         context.setOriginalInputFile(config.getScriptFileName());
-                        exitCode = engine.eval(loadBootSource(
-                                //language=ruby
```

Comment:
Right, could you add them to the new String literals? I can't easily test with IntelliJ.
How does it help? Does it notice there is a caller of Truffle::Boot.main then?